ledger/src/lib.rs

```diff
@@ -109,11 +118,31 @@ pub struct Ledger<N: Network, C: ConsensusStorage<N>> {
     genesis_block: Block<N>,
     /// The current epoch hash.
     current_epoch_hash: Arc<RwLock<Option<N::BlockHash>>>,
-    /// The current committee.
+    /// The committee resulting from all the bonding and unbonding transactions in the blockchain.
```

Comment:
The committee is updated not just from bonding/undbonding, but also staking.

```suggestion
    /// The committee resulting from all the on-chain staking activity.
```